app/models/ems_refresh/save_inventory_container.rb

```diff
@@ -404,14 +404,9 @@ def save_custom_attribute_attribute_inventory(entity, attribute_name, hashes, ta
     return if hashes.nil?
 
     entity.send(attribute_name).reset
-    deletes = if target.kind_of?(ExtManagementSystem)
-                :use_association
-              else
-                []
-              end
 
     save_inventory_multi(entity.send(attribute_name),
-                         hashes, deletes, [:section, :name])
+                         hashes, :use_association, [:section, :name])
```

Comment:
@djberg96 - l heard on SU that you are working on refactoring the shared code referred to here. Can you include the code used in this PR?